docs/blog/september-contributions-showcase.md

```diff
@@ -0,0 +1,120 @@
+
+Big wins for the e18e community!
+
+Thanks to [@benjaminmccann](https://x.com/benjaminmccann) and talentlessguy for contributing _many_ v3 to v4 upgrades already!
+
+## fdir / tinyglobby
+
+`fdir` was in last month's showcase but has earned another mention thanks to the author ([thecodrr](https://x.com/thecodrr)) landing various community contributed PRs and features.
+
+This is a great example of where an author has joined the community and worked closely with it to further their library, and provide some much needed fixes and features.
+
+In version [6.4.0](https://github.com/thecodrr/fdir/releases/tag/v6.4.0), we saw two important changes:
+
+- Ability to exclude symlinks
+- Ability to bring your own glob library (e.g. [zeptomatch](https://github.com/fabiospampinato/zeptomatch))
+
+These two changes have helped the community migrate even more high level tools and frameworks across.
+
+Similarly, tinyglobby has shipped a few new versions. Particularly it is now able to determine if a string is a glob or not. This is super useful for migrating more projects to it.
+
+Thanks to pralkarz, [@benjaminmccann](https://x.com/benjaminmccann) and [@superchupudev](https://x.com/superchupu) for already migrating so many projects to these two libraries!
+
+## tinyexec migrations
+
+The community has been working wonders migrating countless projects from `execa` and other heavyweight libraries to the super lean [tinyexec](https://github.com/tinylibs/tinyexec/).
+
+For example, thanks to pralkarz, [vitest](https://github.com/vitest-dev/vitest) has already moved over!
+
+There are many more PRs currently in progress to do just the same :pray: Great work by the community here.
+
+# Contributions / Improvements
+
+## storybook
+
+The [storybook](https://storybook.js.org/) team have been working closely with the e18e community for some time now, dedicating huge amounts of time to improving the performance and install footprint for everyone.
+
+This month, they've kept up the pace! Just a few examples of what has landed:
+
+- Migrated from `chalk` to `picocolors`
+- Migrated from `fs-extra` to built in `fs` functions
+- Removed `handlebars` and simplified templating internally
+- Removed `prettier` from the core bundle (saved **4MB** on the install size)
+- Replaced `lodash` with `es-toolkit` (more huge savings, especially with tree shaking)
+
+On top of this, the team has actually started a full project focusing on [reducing the install footprint](https://github.com/storybookjs/storybook/issues/29038) in collaboration with the e18e community.
+
+For anyone interested in helping out in the e18e space, this issue a great way to learn some of the techniques used to track down bundle/install size issues.
+
+As a bonus, they even integrated bundle size analysis into their CI so all new PRs can be compared with the main branch. Would love to see that in more projects!
```

Comment:
Could be cool to add a link to where this was setup or some info about it to help others who might want to do the same

Reply:
I don't know if this is _too_ detailed:

```suggestion
As a bonus, they even integrated visual bundle size analysis into their CI so all new PRs can be compared with the main branch. They did this by:

1. Generating [esbuild metafiles](https://esbuild.github.io/api/#build-metadata) for all of their packages every time they're built
2. Importing those metafiles in a story in the repository's internal Storybook.
3. That story sneakily passes the chosen metafile into the [esbuild Bundle Size Analyzer website](https://esbuild.github.io/analyze/) via a query parameter, which then gives a nice visual overview of the bundled output.

You can [play around with the result in their Storybook](https://next--635781f3500dd2c49e189caf.chromatic.com/?path=/story/bench--es-build-analyzer&args=metafile:builder-vite__metafile) and navigate through the different packages via the Controls panel at the bottom.

Would love to see that in more projects! You can get inspiration from [the PR](https://github.com/storybookjs/storybook/pull/29117) that implemented the work. Storybook isn't necessary for any of this to work (it was just a convenient entry point for them) - all of this can be achieved with cleverly crafted URLs and some patience. ✨
```

Happy to reword it if necessary, to make it more approachable for the non-Storybook audience.

Reply:
I think it might be. It almost justifies it's own blog post from your end I think 👀

If you could think of a good way to make it a little more of a summary, that would be super helpful (as a suggestion again so I can commit from the GitHub UI 🙏)

Reply:
Is this better?

```suggestion
As a bonus, they even integrated visual bundle size analysis into their CI so all new PRs can be compared with the main branch.
They did this by generating [esbuild metafiles](https://esbuild.github.io/api/#build-metadata) for all of their packages every time they're built and storing links to the [esbuild Bundle Size Analyzer website](https://esbuild.github.io/analyze/) with those metafiles embedded.
The final result is a visual overview of all package's bundle, that you can [play around with here](https://next--635781f3500dd2c49e189caf.chromatic.com/?path=/story/bench--es-build-analyzer&args=metafile:builder-vite__metafile).

Would love to see that in more projects! You can get inspiration from [the PR](https://github.com/storybookjs/storybook/pull/29117) that implemented the work.
```

Reply:
Looks good to me! 🥳